woocommerce/woocommerce-ios
# Dashboard charts: fix double minus signs in the y-axis for negative values

Fixes #2431 

## Changes

In a [previous PR](https://github.com/woocommerce/woocommerce-ios/pull/2131/files) that I made, the original  L87-89 were removed from `CurrencyFormatter` without the knowledge of breaking this use case in the y-values in the dashboard charts. 🙇🏻‍♀️

In this PR, these lines were added back with a unit test case for the dashboard charts usage (this test case would fail without the changes in `CurrencyFormatter`).

## Testing

Prerequisite: the store has some refunds in the past year, so that you can reproduce the double negative sign issue in the Dashboard charts in `develop` or since release 4.1.

- Launch the app with stats v4
- Tap into each dashboard tab (Today/This Week/This Month/This Year) --> any value in the y-axis should only show at most one minus sign
- Go to Settings > Experimental Features, and turn off "Improved stats" switch
- Tap into each dashboard tab (Days/Weeks/Months/Years) --> any value in the y-axis should only show at most one minus sign

## Example screenshots

(Please note the negative value in the y-axis!)

before | after
-- | --
![Simulator Screen Shot - iPhone 11 - 2020-06-15 at 13 09 52](https://user-images.githubusercontent.com/1945542/84621049-2d23a300-af0c-11ea-8164-6d3d4a81e7f5.png) | ![Simulator Screen Shot - iPhone 11 - 2020-06-15 at 13 10 37](https://user-images.githubusercontent.com/1945542/84621057-31e85700-af0c-11ea-8e08-09aecc11a225.png)


Update release notes:
